```
[ 19%] Building CXX object src/server/game/CMakeFiles/game.dir/AI/CreatureAIRegistry.cpp.o
/home/runner/work/mod-who-logged/mod-who-logged/modules/mod-who-logged/src/who_logged.cpp:57:9: fatal error: misleading indentation; statement is not part of the previous 'if' [-Wmisleading-indentation]
   57 |         if (WorldSession* session = player->GetSession())
      |         ^
/home/runner/work/mod-who-logged/mod-who-logged/modules/mod-who-logged/src/who_logged.cpp:54:5: note: previous statement is here
   54 |     if (!sConfigMgr->GetOption<bool>("PlayerLogoutAnnounce", true))
      |     ^
1 error generated.
gmake[2]: *** [modules/CMakeFiles/modules.dir/build.make:93: modules/CMakeFiles/modules.dir/mod-who-logged/src/who_logged.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1473: modules/CMakeFiles/modules.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
```

--- a/src/who_logged.cpp
+++ b/src/who_logged.cpp
@@ -53,13 +53,13 @@ void WhoLoggedAnnounce::OnPlayerLogout(Player* player)
 {
     if (!sConfigMgr->GetOption<bool>("PlayerLogoutAnnounce", true))
         return;
-	
+        
         if (WorldSession* session = player->GetSession())
         {
             std::string playerIP = session->GetRemoteAddress();
             uint32 playerAccountID = session->GetAccountId();
             std::string playerName = player->GetName();
-	
+                
             LOG_INFO("module", "Player '{}' has logged out : IP '{}' : AccountID '{}'", playerName.c_str(), playerIP.c_str(), playerAccountID);
         }
 }

--- a/src/who_logged.cpp
+++ b/src/who_logged.cpp
@@ -54,12 +54,12 @@ void WhoLoggedAnnounce::OnPlayerLogout(Player* player)
     if (!sConfigMgr->GetOption<bool>("PlayerLogoutAnnounce", true))
         return;
         
-        if (WorldSession* session = player->GetSession())
-        {
-            std::string playerIP = session->GetRemoteAddress();
-            uint32 playerAccountID = session->GetAccountId();
-            std::string playerName = player->GetName();
-                
-            LOG_INFO("module", "Player '{}' has logged out : IP '{}' : AccountID '{}'", playerName.c_str(), playerIP.c_str(), playerAccountID);
-        }
+    if (WorldSession* session = player->GetSession())
+    {
+        std::string playerIP = session->GetRemoteAddress();
+        uint32 playerAccountID = session->GetAccountId();
+        std::string playerName = player->GetName();
+
+        LOG_INFO("module", "Player '{}' has logged out : IP '{}' : AccountID '{}'", playerName.c_str(), playerIP.c_str(), playerAccountID);
+    }
 }

